src/core/providers/proxy.ts

```diff
@@ -4,9 +4,9 @@ export const proxyRpcEndpoint = (endpoint: string, chainId: ChainId) => {
   if (
     endpoint !== 'http://127.0.0.1:8545' &&
     endpoint !== 'http://localhost:8545' &&
-    !endpoint.includes('http://10.') &&
-    !endpoint.includes('http://192.168') &&
-    !endpoint.match(/http:\/\/172.(1[6-9]|2[0-9]|3[0-1])./)
+    !endpoint?.includes('http://10.') &&
+    !endpoint?.includes('http://192.168') &&
+    !endpoint?.match(/http:\/\/172.(1[6-9]|2[0-9]|3[0-1])./)
```

Comment:
i'm wondering if there's a bug happening because of this and typecheck would just fix the crash

https://github.com/rainbow-me/browser-extension/blob/master/src/core/wagmi/createWagmiClient.ts#L51

we're checking whether the original rpc is present or not, maybe we have some bad params for a chain?

Reply:
In this case `getOriginalRpcEndpoint` is returning `{ http: undefined }` for blast.